jetson/percep/artag_detector.cpp

```diff
@@ -127,4 +129,28 @@ cv::aruco::drawDetectedMarkers(rgb, corners, ids);
 
 double TagDetector::getAngle(float xPixel, float wPixel){
     return atan((xPixel - wPixel/2)/(wPixel/2)* tan(fieldofView/2))* 180.0 /PI;
+}
+
+void TagDetector::tagfound(rover_msgs::Target* arTags, pair<Tag, Tag> &tagPair, int tag, int buffer, Mat &depth_img, Mat &src){
```

Comment:
You have `* ` (space after `*`) and ` &` (space before `&`). Let's make this consistent.